From fcf976d5fcfb81d14ea0ec9b9909bc9b8dc52a81 Mon Sep 17 00:00:00 2001
From: Michael Catanzaro <mcatanzaro@redhat.com>
Date: Fri, 19 Nov 2021 08:27:28 -0600
Subject: [PATCH] flatpak: attempt to fix build on i686

There may be more problems, but this should fix the first build failure.
---
 src/plugins/flatpak/daemon/ipc-flatpak-service-impl.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/plugins/flatpak/daemon/ipc-flatpak-service-impl.c b/src/plugins/flatpak/daemon/ipc-flatpak-service-impl.c
index 0f2da7456..6e9d82732 100644
--- a/src/plugins/flatpak/daemon/ipc-flatpak-service-impl.c
+++ b/src/plugins/flatpak/daemon/ipc-flatpak-service-impl.c
@@ -597,7 +597,7 @@ is_known_worker (GTask        *task,
     }
 
 finish:
-  g_debug ("RuntimeIsKnown => (%d, %"G_GSIZE_FORMAT")", found, download_size);
+  g_debug ("RuntimeIsKnown => (%d, %"G_GINT64_FORMAT")", found, download_size);
   ipc_flatpak_service_complete_runtime_is_known (g_task_get_source_object (task),
                                                  g_steal_pointer (&state->invocation),
                                                  found,
-- 
GitLab

